src/util/Collection.js

```diff
@@ -28,6 +28,16 @@ class Collection extends Map {
     Object.defineProperty(this, '_keyArray', { value: null, writable: true, configurable: true });
   }
 
+  /**
+   * Determines whether the passed values is a {@link Collection}.
+   * @param {*} c The value to be checked.
+   * @returns {boolean} `true` if the value is a {@link Collection}; otherwise `false`.
+   * @static
+   */
+  static isCollection(c) {
```

Comment:
Does the param have to be `c`?

We already have `fn equals` which uses `collection`, no real reason to be cheap on letters here, right?

Reply:
Since the param type could be anything I thought that could possibly cause some confusion. Should I use `arg` instead or just go with `collection` anyways?